### PR TITLE
Add translations of the title of the image component

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

Related to https://github.com/vtex-apps/store-image/pull/9

#### How should this be manually tested?

[Workspace](https://imagetitle--storecomponents.myvtex.com/admin/cms/site-editor/about-us) hover the image

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [X] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: messages/context.json [hunk 1]
```diff
@@ -125,6 +125,8 @@
   "admin/editor.image.description": "Show any image",
   "admin/editor.image.src.title": "Image",
   "admin/editor.image.alt.title": "Alternative text",
+  "admin/editor.image.title.title": "Title of the image",
```
Comment (victorhmp) [inline messages/context.json]:
```suggestion
  "admin/editor.image.title.title": "Image title",
```

File: messages/context.json [hunk 2]
```diff
@@ -125,6 +125,8 @@
...
+  "admin/editor.image.title.description": "Title to be shown when you hover the image",
```
Comment (victorhmp) [inline messages/context.json]:
```suggestion
  "admin/editor.image.title.description": "Title to be shown on hover",
```

File: messages/en.json [hunk 3]
```diff
@@ -124,6 +124,8 @@
   "admin/editor.image.title": "Image",
   "admin/editor.image.description": "Show any image",
   "admin/editor.image.src.title": "Image",
+  "admin/editor.image.title.title": "Title of the image",
```
Comment (victorhmp) [inline messages/en.json]: Same from above

File: messages/en.json [hunk 4]
```diff
@@ -124,6 +124,8 @@
...
+  "admin/editor.image.title.description": "Title to be shown when you hover the image",
```
Comment (victorhmp) [inline messages/en.json]: Same from above

File: messages/en.json [hunk 5]
```diff
@@ -124,6 +124,8 @@
   "admin/editor.image.title": "Image",
   "admin/editor.image.description": "Show any image",
   "admin/editor.image.src.title": "Image",
+  "admin/editor.image.title.title": "Image title",
+  "admin/editor.image.title.description": "Title to be shown on hovere",
```
Comment (klzns) [inline messages/en.json]:
```suggestion
  "admin/editor.image.title.description": "Title to be shown on hover",
```

**Pull request merged**